src/app/components/Header/index.jsx

```diff
@@ -1,19 +1,6 @@
 import React from 'react';
 import styled from 'styled-components';
-import {
-  C_POSTBOX,
-  C_WHITE,
-  FF_NEWS_SANS_REG,
-  GEL_SPACING_DBL,
-} from '../../lib/constants/styles';
-
-const Link = styled.a`
-  color: ${C_WHITE};
-  font-family: ${FF_NEWS_SANS_REG};
-  font-size: 2.2em;
-  text-decoration: none;
-  text-transform: uppercase;
-`;
+import { C_POSTBOX, GEL_SPACING_DBL } from '../../lib/constants/styles';
```

Comment:
NB I deliberately didn't rmv the other constants from their source file. Even where they're unused I thought we would imminently want them in the library.